src/personality-chart-renderer.js

```diff
@@ -524,7 +526,7 @@ var renderChart = function() {
       if (text && text.length > 0) {
         var position = d3.select(this).attr('position-in-sector'); // 'inner' or 'outer'
         var frac = position === 'center' ? 0.5 : position === 'outer' ? 2 / 3 : 1 / 3;
-        var sector_length = (d.y + d.dy * frac) * d.dx;
+        var sector_length = (d.y1 * frac) * (d.x1 - d.x0);
```

Comment:
This changes the meaning of the equation due to order of operations.  Should be:
```
var sector_length = (d.y0 + ((d.y1 - d.y0) * frac) * (d.x1 - d.x0);
```